staging/src/k8s.io/apiserver/pkg/authentication/token/cache/stats.go

```diff
@@ -42,22 +42,23 @@ var (
 		},
 		[]string{"status"},
 	)
-	fetchCount = metrics.NewGaugeVec(
-		&metrics.GaugeOpts{
+	fetchCount = metrics.NewCounterVec(
```

Comment:
Is it OK to change an existing metric from a gauge to a counter?

Reply:
I don't see how it could be --- the metrics aggregator will be seeing some copies that claim to be gauges and some copies that claim to be counters.

Reply:
It's not used yet. I just added it a few weeks ago.